driver/src/test/functional/com/mongodb/client/Fixture.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2018 MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb.client;
+
+import com.mongodb.ConnectionString;
+
+/**
+ * Helper class for the acceptance tests.  Considering replacing with MongoClientTestBase.
```

Comment:
Mostly copy and paste from Fixture in driver-legacy.  Just notice this out-dated comment.  Will fix.

Reply:
Note: Minor todo.